src/v2/ion/ui-schema/ion-boolean-handler.js

```diff
@@ -42,6 +58,8 @@ const createUiSchemaForBoolean = (ionFormField, remediationForm) => {
     const type = remediationForm.name;
 
     return {type, scopes, options: ionFormField.options};
+  } else if (remediationForm.name === FORMS.CONSENT_GRANULAR) {
```

Comment:
Please add unit tests for these additions

Reply:
BaseForm defines a method `getUISchema` - this may be a more appropriate place for these types of customizations which only apply to a single view

Reply:
I added tests in https://github.com/okta/okta-signin-widget/pull/2545/files#diff-04ac2de0f3ff03b3f319bc33337f8501bf4ec4f5c0ce47bc313e57603fca3f3f, are there other types of tests that I need to add?
And updated to override `getUISchema` in https://github.com/okta/okta-signin-widget/pull/2545/commits/4a14d5385fc4b38622e41f41c58f88beb7b18c88